.gitlab-ci.yml

```diff
@@ -65,7 +65,7 @@ before_script:
 
 include:
   - ci/environment.yml
-  - ci/prebuild.yml
+#  - ci/prebuild.yml
```

Comment:
It is in the description but I admit it was hidden a bit in the text.

> prestyle is currently commented out because of #934, although it might be fixed by removing the pipenv update most likely.